src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -62,17 +67,25 @@ export default function AppointmentsPageV2() {
   const location = useLocation();
   const [hasTypeChanged, setHasTypeChanged] = useState(false);
   const showScheduleButton = useSelector(state => selectFeatureRequests(state));
+  const featureStatusImprovement = useSelector(state =>
+    selectFeatureStatusImprovement(state),
+  );
+  const pendingAppointments = useSelector(state =>
+    selectPendingAppointments(state),
+  );
   const {
     dropdownValue,
     subPageTitle,
     subHeading,
   } = getDropdownValueFromLocation(location.pathname);
 
+  const [count, setCount] = useState(0);
   useEffect(
     () => {
+      if (featureStatusImprovement) pageTitle = 'Your appointments';
       document.title = `${subPageTitle} | ${pageTitle} | Veterans Affairs`;
     },
-    [subPageTitle],
+    [subPageTitle, featureStatusImprovement],
```

Comment:
I think we can exclude `featureStatusImprovement` from re-rending the page since this is a toggle flag boolean that remains static.
```suggestion
    [subPageTitle],
```

Reply:
ESLint was complaining when missing.